tests/ignite/metrics/test_classification_report.py

```diff
@@ -0,0 +1,297 @@
+import json
+import os
+
+import pytest
+import torch
+
+import ignite.distributed as idist
+from ignite.engine import Engine
+from ignite.metrics.classification_report import ClassificationReport
+
+
+def _test_integration_binary(device):
+
+    rank = idist.get_rank()
+    torch.manual_seed(12)
+
+    def _test(metric_device):
+
+        n_iters = 80
+        s = 16
+        offset = n_iters * s
+        y_true = torch.randint(0, 2, size=(offset * idist.get_world_size(),)).to(device)
+        y_preds = torch.randint(0, 2, size=(offset * idist.get_world_size(),)).to(device)
+        y_true_unflat = _unflatten_binary(y_true)
```

Comment:
I do not quite understand why input data is of that form. At first, it is not a binary case but 2 classes multi-class...